app/services/jwt/encode_service.rb

```diff
@@ -29,7 +29,7 @@ def claims
       now = Time.zone.now
       {
         aud: Settings.teacher_training_api.audience,
-        exp: (now + 5.minutes).to_i,
+        exp: (now + 6.hours).to_i,
```

Comment:
`6.hours.from_now` is beautifuller 😆

Reply:
i know but the ticks will be off for the issue

Reply:
https://github.com/DFE-Digital/publish-teacher-training/blob/c12eedf7bc26f4fd20d6173ad68399b5fc5df733/app%2Fservices%2Fjwt%2Fencode_service.rb#L33